src/components/OpenWith/OpenWithButton.js

```diff
@@ -14,7 +14,7 @@ import ICON_FILE_MAP from './IconFileMap';
 import messages from '../messages';
 
 type Props = {
-    displayIntegration: ?Integration | ?Object,
+    displayIntegration?: ?Integration | ?Object,
```

Comment:
Lots of ? 
are all needed?

Reply:
I think I can probably get rid of that object, I'll play around with it.